l10n_ch_payment_slip/tests/test_payment_slip.py

```diff
@@ -1,15 +1,19 @@
-# -*- coding: utf-8 -*-
-# © 2014-2016 Camptocamp SA
+# Copyright 2014-2017 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 import time
 import re
 import odoo.tests.common as test_common
-from odoo.report import render_report
 
 
 class TestPaymentSlip(test_common.TransactionCase):
     _compile_get_ref = re.compile(r'[^0-9]')
 
+    def setUp(self):
+        super(TestPaymentSlip, self).setUp()
+        self.report1slip_from_inv = self.env.ref(
+            'l10n_ch_payment_slip.one_slip_per_page_from_invoice',
+        )
+
     def make_bank(self):
```

Comment:
Would be great to have these records as demo data as it's impossible to test functionally without it.